composer.json

```diff
@@ -14,7 +14,7 @@
     ],
     "require": {
         "php": ">=5.3.1",
-        "symfony/process": "~2.1|~3.0"
+        "symfony/process": "~4"
```

Comment:
this should at least allow 3.4 too, as it also has the array syntax for the Process class

Reply:
I thought that as well, but there are conflicts with older versions.